tests/unittest.c

```diff
@@ -218,6 +218,8 @@ int main(int argc, char *argv[])
 	size_t i, test_num = 0;
 	int c;
 
+	setup_color(COLOR_OFF);
+
```

Comment:
I don't want to handle it like this.  And unittest has an option (-n) to turn off color.

Instead, I think `get_enum_string()` should return a plain string without color code.  It's the job of `get_argspec_string()` to make up the fancy colors.  Maybe it's a bit more complex but I think it should work like that.